go.mod

```diff
@@ -28,38 +25,29 @@ require (
 	github.com/coreos/go-semver v0.3.0 // indirect
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
 	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
-	github.com/eapache/go-resiliency v1.2.0 // indirect
+	github.com/dapr/dapr v0.1.0
```

Comment:
component-contrib should not have the dependency of dapr.

Reply:
Yep, forgot to replace call to the function that was moved into components-contrib.